client/my-sites/upgrades/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -118,6 +120,26 @@ class EditContactInfoFormCard extends React.Component {
 		}
 	}
 
+	hasEmailChanged() {
+		return this.props.contactInformation.email === formState.getFieldValue( this.state.form, 'email' );
+	}
+
+	getCurrentEmails() {
+		const currentEmail = this.props.contactInformation.email,
+			wpcomEmail = this.props.currentUser.email;
+
+		if ( currentEmail === wpcomEmail ) {
+			return currentEmail;
+		}
+
+		return this.props.translate( '%(currentEmail)s (additionally to %(wpcomEmail)s)',
+			{
+				args: { currentEmail, wpcomEmail },
+				context: 'List of emails the WHOIS confirmation email is sent to'
```

Comment:
I don't see this copy in the screenshots above. Do you have a screenshot of where it shows up?